std/conv.d

```diff
@@ -230,7 +221,7 @@ template to(T)
  * When converting strings _to numeric types, note that the D hexadecimal and binary
  * literals are not handled. Neither the prefixes that indicate the base, nor the
  * horizontal bar used _to separate groups of digits are recognized. This also
- * applies to the suffixes that indicate the type.
+ * applies _to the suffixes that indicate the type.
```

Comment:
You hit a wall here. Don't merge two things in one PR. End of story. Waiting for you to amend.
Open another PR for the ddoc.